### PR TITLE
feat: added step 4 for onboarding in dtrader v2

### PR DESCRIPTION
**Pull request by vinu-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

File: packages/trader/src/AppV2/Containers/Trade/trade.tsx [hunk 1]
```diff
@@ -88,17 +88,19 @@ const Trade = observer(() => {
...
-                        </section>
+                        <div className='trade__chart-tooltip'>
+                            <section className='trade__chart' style={{ height: dynamic_chart_height }} ref={chart_ref}>
+                                <TradeChart />
+                            </section>
+                        </div>
                         {isAccumulatorContract(contract_type) && <AccumulatorStats />}
                     </div>
-                    <div className='trade__bottom'>
+                    <span className='trade__bottom'>
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Containers/Trade/trade.tsx:98]: Does it complain, that in span we have block tags?

File: packages/trader/src/AppV2/Components/OnboardingGuide/guide-container.tsx [hunk 2]
```diff
@@ -90,7 +116,8 @@ const GuideContainer = ({ should_run, onFinishGuide }: TGuideContainerProps) =>
                     borderRadius: 'unset',
                 },
             }}
-            run={should_run}
+            stepIndex={step_index}
+            tooltipComponent={props => <GuideTooltip {...props} setStepIndex={setStepIndex} />}
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Components/OnboardingGuide/guide-container.tsx:120]: Nice

File: packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx [hunk 3]
```diff
@@ -1,42 +1,75 @@
...
+import { useSwipeable } from 'react-swipeable';
 
-const GuideTooltip = ({ isLastStep, primaryProps, skipProps, step, tooltipProps }: TooltipRenderProps) => (
-    <div {...tooltipProps} className='guide-tooltip__wrapper'>
-        <div>
-            {step.title && (
-                <div className='guide-tooltip__header'>
-                    <CaptionText bold color='var(--component-textIcon-inverse-prominent)'>
-                        {step.title}
+export interface GuideTooltipProps extends TooltipRenderProps {
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx:8]: In dtrader we usually use types, not interfaces. For me looks good, but we might get extra comments in PR

Comment (kate-deriv) [reply packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx:8]: Up to you

File: packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx [hunk 4]
```diff
@@ -1,42 +1,75 @@
...
+        preventDefaultTouchmoveEvent: true,
+        trackTouch: true,
+        trackMouse: true,
+    });
+
+    if (step.title === 'scroll-icon') {
+        return (
+            <div {...swipe_handlers} className='guide-tooltip__wrapper-scroll'>
+                <LabelPairedChevronsUpXlBoldIcon
+                    fill='var(--core-color-solid-slate-50)'
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx:27]: we can also move fill inside of styles for guide-tooltip--bounce class. Up to you, it won't make big difference

File: packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx [hunk 5]
```diff
@@ -1,42 +1,75 @@
...
+                    className='guide-tooltip--bounce'
+                />
+                <Localize i18n_default_text='Swipe up to see the chart' />
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx:30]: We can use some typography component here (around <Localize/>) in order to get rid of font styles in packages/trader/src/AppV2/Components/OnboardingGuide/onboarding-guide.scss

File: packages/trader/src/AppV2/Containers/Trade/trade.tsx (hunk 1 above)
Comment (kate-deriv) [reply packages/trader/src/AppV2/Containers/Trade/trade.tsx:98]: I mean console errors?

File: packages/trader/src/AppV2/Components/OnboardingGuide/onboarding-guide.scss [hunk 6]
```diff
@@ -29,6 +39,22 @@
...
+        100% {
+            transform: translateY(0);
+        }
+        50% {
+            transform: translateY(-10px);
+        }
+    }
+
+    &--bounce {
+        height: 64px;
```
Comment (kate-deriv) [inline packages/trader/src/AppV2/Components/OnboardingGuide/onboarding-guide.scss:54]: Will we use px directly or quill tokens for size?

File: packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx (hunk 5 above)
Comment (vinu-deriv) [reply packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx]: but had to classname for typography component

File: packages/trader/src/AppV2/Containers/Trade/trade.tsx (hunk 1 above)
Comment (vinu-deriv) [reply packages/trader/src/AppV2/Containers/Trade/trade.tsx]: I have made this change

**Pull request merged**